python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -478,6 +486,16 @@ class Meta:
         description="Retrieve the auto materialization evaluation records for all assets.",
     )
 
+    autoMaterializeTicks = graphene.Field(
```

Comment:
should we have instigator state for auto-materialize?  e.g. control turning on/off via that?

then we'd just have a resolver for the state and then you could use the existing schema to fetch ticks?

I guess some of the required repository fields are a little awkward there